lib/args.js

```diff
@@ -0,0 +1,101 @@
+"use strict";
```

Comment:
Please don't commit lib files in the repo

Reply:
Sure thing, I was using this branch for testing that is why I committed the lib folder.